subsys/bluetooth/host/Kconfig

```diff
@@ -335,6 +335,17 @@ config BT_SCAN_WITH_IDENTITY
 	  disclosing local identity information. However, if the use case
 	  requires disclosing it then enable this option.
 
+config BT_DEVICE_NAME_STORAGE
```

Comment:
I'd call it `BT_DEVICE_NAME_MAX` since we haven't (at least so far) had the convention of using `STORAGE` for the upper limit of something.

Reply:
Ok